pkg/distro/fedora/distro.go

```diff
@@ -397,7 +397,6 @@ type distribution struct {
 	releaseVersion     string
 	modulePlatformID   string
 	ostreeRefTmpl      string
-	isolabelTmpl       string
```

Comment:
I'm wondering if it wouldn't be good to make the same changes in RHEL distros, especially for consistency and not diverging in structs used by each distro.

Reply:
I can make the same change in RHEL, just wanted to make sure before I touched RHEL :)

Reply:
Do we want to do RHEL in this PR or in a separate PR?

Reply:
Up to you. I don't have any strong preference, as long as it is eventually done.